anarchy-runner/anarchy-runner.py

```diff
@@ -224,28 +224,27 @@ def setup_python_venv(self, anarchy_run):
         virtual_env = self.ansible_private_dir + '/pythonvenv-' + requirements_md5
         requirements_file = virtual_env + '/requirements.txt'
         if not os.path.exists(virtual_env):
-            venv.EnvBuilder(system_site_packages=True, with_pip=True).create(virtual_env)
+            # Copy virtualenv from /opt/app-root
+            os.makedirs(virtual_env)
+            shutil.copy('/opt/app-root/pyvenv.cfg', virtual_env + '/pyenv.cfg')
+            shutil.copytree('/opt/app-root/bin', virtual_env + '/bin')
+            shutil.copytree('/opt/app-root/include', virtual_env + '/include')
+            shutil.copytree('/opt/app-root/lib', virtual_env + '/lib')
+            os.symlink('lib', virtual_env + '/lib64')
+
+            # Write requirements.txt
```

Comment:
The way the virtual env is created had to change to support the Python 3.8 base image. The Python installation in /opt/app-root is itself a virtualen, so using system site packages did not work. This approach copies the base virtualenv to get a similar effect.